CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
+
+## [Unreleased]
+
+## [v0.4.0] - 2022-07-01
+
+### Added
+
+- Added a simple queueing interface to the `execution` package
+- Updated the `inmemory` state package to implement the new queue package
+- Added distributed waitgroups to the `state.Manager` interface.
+
+### Changed
+
+- Changed the `state.Manager` interface to record driver responses directly
+- Changed the mechanisms for the dev server and `inngest run` to use distributed
+  waitgroups when running functions, and to use the new queue interface.
+
```

Comment:
Should we have a section in future change logs highlighting if there are breaking changes or not? e.g. 
```md 
### Breaking Changes
n/a
```

Reply:
Updated!